### PR TITLE
feat(jest): 22% coverage for MetadataManager

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

A work in progress. Improvements soon to be made, right now we're hit by two errors:

- Textile's Query
- Textile's publickey (encrypt)

**Which issue(s) this PR fixes** 🔨

AP-1338

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: libraries/Textile/MetadataManager.test.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  test('', async () => {
+    ThreadID.fromString.mockImplementationOnce(() => new Uint8Array([]))
+    const result = await konstruktor.init()
+    expect(result).toBeUndefined() // Dummy assertion, we'll replace it with what the result returns
+  })
+  test.skip('updateMetadata with threadID', async () => {
+    // Throws error
+    // TypeError: publicKey.encrypt is not a function
+    // at MetadataManager._encodeBody (libraries/Textile/MetadataManager.ts:83:23)
+    // at MetadataManager.updateMetadata (libraries/Textile/MetadataManager.ts:168:29)
```
Comment (stavares843) [inline libraries/Textile/MetadataManager.test.ts:67]: if the test is skipped, do we need this commented out?

File: libraries/Textile/MetadataManager.test.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    ThreadID.fromString.mockImplementationOnce(() => new Uint8Array([]))
+    const result = await konstruktor.updateMetadata({
+      to: `0xrecipientkey`,
+      from: `0xsenderkey`,
+      metadata: { note: 'note' },
+    })
+    expect(result).toBe({}) // Dummy assertion, we'll replace it with what the result returns
+  })
+  test.skip('getMetadata', async () => {
+    // Throws error
```
Comment (stavares843) [inline libraries/Textile/MetadataManager.test.ts:77]: if the test is skipped, do we need this commented out?

**Review by stavares843 — COMMENTED** [review]
added minor comments

File: libraries/Textile/MetadataManager.test.ts (hunk 1 above)
Comment (drepram) [reply libraries/Textile/MetadataManager.test.ts:67]: Yes, as to give clarity when we go back to this code and try to resolve this error.

File: libraries/Textile/MetadataManager.test.ts (hunk 2 above)
Comment (drepram) [reply libraries/Textile/MetadataManager.test.ts:77]: Gavin MJ

Comment (drepram) [reply libraries/Textile/MetadataManager.test.ts:77]: Yes, as to give clarity when we go back to this code and try to resolve this error.

File: libraries/Textile/MetadataManager.test.ts (hunk 1 above)
Comment (stavares843) [reply libraries/Textile/MetadataManager.test.ts:67]: thanks for clarifying 🔨

File: libraries/Textile/MetadataManager.test.ts (hunk 2 above)
Comment (stavares843) [reply libraries/Textile/MetadataManager.test.ts:77]: thanks for clarifying 🔨

**Review by stavares843 — APPROVED** [review]

**Pull request merged**